documents/entity/model.go

```diff
@@ -267,7 +241,7 @@ func (e *Entity) PrepareNewVersion(old documents.Model, data *cliententitypb.Ent
 	}
 
 	oldCD := old.(*Entity).CoreDocument
-	e.CoreDocument, err = oldCD.PrepareNewVersion(collaborators, true, compactPrefix())
+	e.CoreDocument, err = oldCD.PrepareNewVersion(collaborators, compactPrefix())
```

Comment:
let's change the `PrepareNewVersion` function definition to 
```
PrepareNewVersion(prefix, collaborators ...)
```

This way its more clear IMO